tools/reactive-controllers/src/FocusGroup.ts

```diff
@@ -230,9 +230,9 @@ export class FocusGroupController<T extends HTMLElement>
         }
         switch (this.direction) {
             case 'horizontal':
-                return code === 'ArrowLeft' || code === 'ArrowRight';
+                return code.startsWith('Arrow');
             case 'vertical':
-                return code === 'ArrowUp' || code === 'ArrowDown';
+                return code.startsWith('Arrow');
```

Comment:
Based on the conversation in the issue this stems from, I'm still not sure we want to flatten _ALL_ usage of this controller or if we want to just leverage `both` in Tabs, so that it can be accurately supported. Can you take a look at leveraging that instead?

If really _nothing_ should be `horizontal` or `vertical`, then that's another question. Do we auto fallback to both like this? And then use Dev Mode warnings to clarify why the functionality isn't what might be expected by the input, or do we leave `horizontal` and `vertical` as is, change all of our consumption to `both`, and then Dev Mode warn that `horizontal` and `vertical` may not deliver the expected screen reader/keyboard experience.

Reply:
That is actually a good point to consider here... I feel though that this should apply to pretty much all the components leveraging the controller right (not just Tabs) as the functionality we are expecting should be consistent? But sure I can try to leverage the both functionality in just the tabs.

Reply:
@Westbrook  Based on the conversation in issue should I make the changes such that this functionality of using both Down/RightArrow and Up/LeftArrow works specifically with tablist only... I'd start by removing the changes from FocusGroup.ts. Let me know if that makes sense?